.github/workflows/release_linux.yml

```diff
@@ -34,14 +47,14 @@ jobs:
 
     - name: Set up Python ${{ env.Python_REQUIRED_VERSION }}
       id: setup-python
-      uses: jmarrec/setup-python@v5
+      uses: jmarrec/setup-python@v5.4.0
       with:
         python-version: ${{ env.Python_REQUIRED_VERSION }}
 
     - name: Setup QtIFW 4.x
       uses: jmarrec/setup-qtifw@v1
       with:
-        qtifw-version: '4.6.1'
+        qtifw-version: '4.8.1'
```

Comment:
Gotta bump qtfiw, the arm64 linux installers for QtIFW were added at 4.7.0

Reply:
Got it.  And we are using your fork because they didn't take your PR to pass arguments down into binarycreator, right?  Or something like that?  Over time it would be really great to get back to mainline actions so we don't have to rely on custom built ones, but I do appreciate your patches to them.